example/main.tf

```diff
@@ -0,0 +1,105 @@
+terraform {
+  required_version = ">= 0.11.2"
+}
+
+variable "name" {
+  type        = "string"
+  description = "Name (unique identifier for app or service)"
+}
+
+variable "namespace" {
+  type        = "string"
+  description = "Namespace (e.g. `cp` or `cloudposse`)"
+}
+
+variable "delimiter" {
+  description = "The delimiter to be used in labels."
+  default     = "-"
+}
+
+variable "stage" {
+  type        = "string"
+  description = "Stage (e.g. `prod`, `dev`, `staging`)"
+}
+
+variable "attributes" {
+  type        = "list"
+  description = "List of attributes to add to label."
+  default     = []
+}
+
+variable "tags" {
+  type        = "map"
+  description = "Map of Tag name/values."
+  default     = {}
+}
+
+variable "region" {
+  type        = "string"
+  description = "AWS region"
+}
+
+variable "github_oauth_token" {
+  description = "GitHub Oauth Token with permissions to access private repositories"
+}
+
+variable "repo_owner" {
+  description = "GitHub Organization or Username."
+}
+
+variable "repo_name" {
+  description = "GitHub repository name of the application to be built and deployed to ECS."
+}
+
+variable "branch" {
+  description = "Branch of the GitHub repository, e.g. master"
+}
+
+provider "aws" {
+  region = "${var.region}"
+}
+
+module "container_definition" {
+  source           = "git::https://github.com/cloudposse/terraform-aws-ecs-container-definition.git?ref=0.1.3"
+  container_name   = "${var.name}"
+  container_image  = "nginx:latest"
+  container_memory = 128
+  container_port   = 80
+
+  log_options = {
+    "awslogs-region" = "${var.region}"
+
```

Comment:
We can delete this extra whitespace. When I tried it, `terraform fmt .` did not add it.